Modules/Options/Options.lua

```diff
@@ -1176,6 +1176,23 @@ function R:PrepareOptions()
 									Rarity.GUI:UpdateText()
 								end,
 							},
+							elements = {
+								order = newOrder(),
+								type = "range",
+								width = "double",
+								name = L["Max Elements"],
```

Comment:
Something tells me this name might be a little unclear, but I guess we'll have to ask others for feedback.